packages/nimble-components/src/tab/styles.ts

```diff
@@ -84,7 +84,7 @@ export const styles = css`
     }
 
     :host(${focusVisible})::before {
-        width: 100%;
+        width: calc(100% - 6px);
```

Comment:
Can you make a private prop for this magic number? Maybe something like `--ni-private-focus-indicator-inset-width` or something?